pages/_middleware.ts

```diff
@@ -1,18 +1,22 @@
 import { NextRequest, NextResponse } from 'next/server'
 
 const BLOCKED_COUNTRIES = ['US', 'BY', 'CU', 'IR', 'IQ', 'CI', 'LR', 'KP', 'SD', 'SY', 'ZW', 'CA']
+const ALLOWED_IPS = ['54.217.161.151']
```

Comment:
Maybe this should be env / vercel deployment variable, might be plan once you test anyways.

Reply:
lets test and then move to env